src/git/frontend_commands.go

```diff
@@ -37,12 +37,14 @@ func (self *FrontendCommands) AbortRebase() error {
 
 // CheckoutBranch checks out the Git branch with the given name in this repo.
 func (self *FrontendCommands) CheckoutBranch(name gitdomain.LocalBranchName) error {
```

Comment:
In my (humble) opinion, you could just add the `merge bool` argument to the existing `CheckoutBranch` method and not create the two additional methods here. Other public methods in this file also have such optional arguments, for example https://github.com/git-town/git-town/blob/main/src/git/frontend_commands.go#L231.

Reply:
done